data/json/monstergroups/zombies.json

```
@@ -364,6 +364,8 @@
     "default": "mon_zombie_prisoner",
     "monsters": [
       { "monster": "mon_zombie_prisoner", "freq": 350, "cost_multiplier": 0 },
+      { "monster": "mon_zombie_prisoner_fat", "freq": 350, "cost_multiplier": 0 },
+      { "monster": "mon_zombie_prisoner_tough", "freq": 350, "cost_multiplier": 0 },
       { "monster": "mon_zombie_static", "freq": 10, "cost_multiplier": 0 }
```

```suggestion
      { "monster": "mon_zombie_prisoner", "freq": 350, "cost_multiplier": 1 },
      { "monster": "mon_zombie_prisoner_fat", "freq": 350, "cost_multiplier": 2 },
      { "monster": "mon_zombie_prisoner_tough", "freq": 350, "cost_multiplier": 2 },
      { "monster": "mon_zombie_static", "freq": 10, "cost_multiplier": 3 }
```
Specific numbers are complete guesses.